core/quantity/src/test/BearingAzimuth.test.ts

```diff
@@ -65,18 +65,18 @@ describe("Bearing format tests:", () => {
 
     const bearingDMS = new Format("BearingDMS");
     await bearingDMS.fromJSON(unitsProvider, bearingDMSJson).catch(() => { });
-    assert.isTrue(bearingDMS.hasUnits);
+    expect(bearingDMS.hasUnits).to.be.true;
```

Comment:
why not toBe(true)? this is nitpicky really,  cuz I switched all my other tests to use jest assertions

Reply:
no reason really. When I began, I started using `.to.be.true` and just kept using that for everything to keep things consistent

Reply:
should we decide on a standard? toBe(true) is a JestAssertion, .to.be.true is chai. Vitest will support both ways, because it has a peerDep on chai and exports chai assertions, but it won't come with async support out of the box. So we would have to handle promises by adding chaiAsPromised explicitly

Reply:
we should decide on a standard for consistency's sake. Should we have a separate PR that does this everywhere in the code? I am in favor of whatever will lead to less deps/code. However, how would we enforce this? Maybe a custom lint rule?

Reply:
i straight up did this to core frontend, david did so for core geometry individually, but you're right that enforcing it would be better. I have no idea how to enforce this though :(

Reply:
After discussion with Rob, there's no need to enforce this. It's a recommendation to use jest assertions instead of chai, in cases where people want to drop chaiAsPromised usage